pytest_catchlog.py

```diff
@@ -139,10 +180,13 @@ def emit(self, record):
 class CatchLogFuncArg(object):
```

Comment:
@The-Compiler Thanks. I thought that "funcarg" and "fixtures" are roughly synonymous, the one just being a modern alternative to the other.

How do I call this class? `CatchLogFixture` / `CatchLog` / `Caplog` / `CaplogObject`?

/cc @eisensheng